### PR TITLE
Add image to cultural change

### PR DESCRIPTION
**Pull request by EstherPlomp**
### Summary

Add one of the older images to the [cultural change section](https://the-turing-way.netlify.app/ethical-research/activism/activism-cultural-change.html)

partially addresses #3331

### CODE REVIEW

File: book/website/ethical-research/activism/activism-cultural-change.md [hunk 1]
```diff
@@ -15,6 +15,14 @@ Here, cultural change is defined as inspiring a change in behaviour in persons/o
 This section will continue to describe where you will encounter cultural change. 
 This is followed by short summaries of works on cultural change, written by Adrienne Marie Brown, William and Susan Bridges and John P. Kotter. 
 
+```{figure} ../../figures/culture-shift.*
+---
+name: culture shift
+alt: "The cartoon says that it is time for a cultural shift: we should value reproducibility as much as the amount of papers published. To illustrate this a scale is depicted, with on the one side a lot of objects, and on the other side only two objects with a shiny star on them. Both side weigh equally."
```
Comment (Arielle-Bennett) [inline book/website/ethical-research/activism/activism-cultural-change.md:21]:
```suggestion
alt: A black, white, and purple cartoon of scales. On the one side are a lot of objects, with a person buried under the pile and peering out, and on the other side only two objects with a shiny star on them, and someone standing next to them with a smile. Both sides weigh equally. The text says 'time for a cultural shift, we should value reproducibility as much as the amount of papers published.'
```

**Review by Arielle-Bennett — COMMENTED** [review]
Added some suggestions for expanding the alt text here 😄 colour scheme and more detail on the image

**Comment by EstherPlomp** [comment]
> Added some suggestions for expanding the alt text here 😄 colour scheme and more detail on the image

Thanks @Arielle-Bennett - that is a lot better :D

**Review by Arielle-Bennett — APPROVED** [review]
Going to approve but up to you if you want to merge or let others weigh in with any other feedback✨

**Comment by EstherPlomp** [comment]
I'm going to merge and go to bed <3 Thanks for the quick feedback and review!

**Pull request merged**